tests/drivers/adc/adc_dma/src/main.c

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright (c) 2020, NXP
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/**
+ * @addtogroup t_driver_adc
+ * @{
+ * @defgroup t_adc_basic test_adc_basic_operations
+ * @}
+ */
```

Comment:
This looks like copy-n-paste? Does it need to be updated for the new sample?

Would it be possible to incorporate these new DMA tests in the existing ADC test suite? In their current form, the look rather NXP ADC16 specific. Perhaps they should go into `tests/drivers` instead?

Reply:
@henrikbrixandersen,  in latest code the comments tag is removed, so let me remove them also.

in-corporate the DMA with NON-DMA is possible, but There will be too many marcos for different settings, and I am afraid this will makes the code hard to read, and thus cost more efforts to maintain. what do you mean go into tests/drivers? it is now in the tests/drivers folder.

Reply:
Oh. I meant it could go into an mcux_adc16 driver specific test folder.

Reply:
@henrikbrixandersen, in this test there are no sopecial settings for adc16, but some adjustment on the buffer usage(use ping pong buffer to compare) and interval constrains(trigger interval shall less than the conversion rate), I guess this will be a common requirements, as there is no other driver supports dma in ADC, shall we keep it as common test as possible?